conf/defaults.ini

```diff
@@ -768,7 +768,10 @@ icon = microsoft
 enabled = false
 allow_sign_up = true
 auto_login = false
+client_authentication =
 client_id = some_client_id
+managed_identity_client_id =
+audience =
```

Comment:
I think this setting name could be a bit misleading. If I knew nothing about this specific feature and just wanted to configure oauth for entra id I would think this is the audience to validate for access / id token we get back after a valid authentication. Maybe we should make it more obvious with the name, something like `managed_identity_client_audience`. WDYT?

Reply:
@kalleep I totally agree with you! Since the `audience` is a direct attribute of the federated credential, let us change the name to `federated_credential_audience`.